src/routes/legislative-process/+page.svelte

```diff
@@ -135,6 +190,10 @@
 		@apply text-text-primary mb-4;
 	}
 
+	h3 {
```

Comment:
I just realized that h1, h2, and h3 have the same style. We can combine it together like

```postcss
h1, h2, h3 {
		@apply text-text-primary mb-4;
}
```